bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -575,6 +573,22 @@ const Whiteboard = React.memo(function Whiteboard(props) {
             zoomSlide(viewedRegionW, viewedRegionH, nextCam.x, nextCam.y, currentPresentationPageRef.current);
           }
         }
+
+        // Check for idle states and persist the batch if there are shapes
+        if (path === 'select.idle' || path === 'draw.idle' || path === 'select.editing_shape') {

```

Comment:
Highlight was not updating, tested this and it worked:

```suggestion
        if (path === 'select.idle' || path === 'draw.idle' || path === 'select.editing_shape' || path === 'highlight.idle') {

```